microsoft-edge/visual-studio-code/elements-for-edge.md

```diff
@@ -15,15 +15,13 @@ By adding the [Elements for Edge](https://marketplace.visualstudio.com/items?ite
 
 ![GIF of the Elements for Edge VS Code extension at work](./media/elements-for-edge.gif)
 
-## Launching Edge From the Elements tab
+## Launching Edge From the Elements tab 
 
-![Elements for Edge tab](./media/elements-for-edge-logo.jpg) 
-
-Navigate to Elements (the icon shown above). Next to where it says Elements for Microsoft Edge: Targets, there is a plus sign that will open the browser for your app. If you selected the about:blank option, you will have to navigate to your web app in the browser for it to appear in the Elements panel in VS Code.
+Navigate to Elements (the icon shown above). Next to where it says Elements for Microsoft Edge: Targets, there is a plus sign that will open the browser for your app. If you selected the *about:blank* option, you will have to navigate to your web app in the browser for it to appear in the Elements panel in VS Code.
```

Comment:
If we removed the image of the icon, do we need to say "(the icon shown above)". Are you planning on eventually adding the annotated screenshot of VS Code here with the extension?